core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -1878,6 +1878,25 @@ public String toString() {
           .setDescription("Maximum concurrency level for the lock pool")
           .setScope(Scope.MASTER)
           .build();
+  public static final PropertyKey MASTER_JOURNAL_INITIAL_REPLAY_ENABLED =
+      new Builder(Name.MASTER_JOURNAL_INITIAL_REPLAY_ENABLED)
+          .setDefaultValue(false)
+          .setDescription("(Experimental) Enable the initial journal replay "
+              + "before joining the quorum in fault tolerant mode "
+              + "when starting the master process. This is added to prevent "
+              + "frequently leadership transition during heavy journal replay stage")
+          .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
+          .setScope(Scope.MASTER)
+          .build();
+  public static final PropertyKey MASTER_JOURNAL_MAX_INITIAL_REPLAY_TIME =
+      new Builder(Name.MASTER_JOURNAL_MAX_INITIAL_REPLAY_TIME)
+          .setDefaultValue("10min")
+          .setDescription("The maximum time to wait for ufs journal catching up "
```

Comment:
I know only the UFS journal implements the initial replay, but could this also apply to raft later? Basically, this description mentions ufs journal, but the property name is not ufs specific.

Reply:
Yeah, change to MASTER_UFS_JOURNAL.
It's interesting that we don't have MASTER_UFS_JOURNAL specific property keys yet.